build.sh

```diff
@@ -49,6 +49,9 @@ fi
 if [[ ! -d "better-enums" ]] ; then
   git clone https://github.com/aantron/better-enums.git --branch 0.11.3 --depth 1
 fi
+if [[ ! -d "pybind" ]] ; then
+  git clone https://github.com/pybind/pybind11.git --branch v2.10 --depth 1
+fi
```

Comment:
I don't know the details, but I'm pretty sure this was placed under the PYBIND option because the people developing the web frontend don't want to download unnecessary things, so I'd suggest putting it back there.